docs/cloudslang_dsl_reference.md

```diff
@@ -349,7 +349,14 @@ inputs:
 The key `do` is a property of a [task](#task) name, a [loop](#loop), or an [async_loop](#async_loop).
 It is mapped to a property that references an [operation](#operation) or [flow](#flow).
 
-Calls an [operation](#operation) or [flow](#flow) and passes in relevant [input](#inputs). The [operation](#operation) or [flow](#flow) may be called by its qualified name using an alias created in the [imports](#imports) parameter.
+Calls an [operation](#operation) or [flow](#flow) and passes in relevant [input](#inputs). 
+
+The [operation](#operation) or [flow](#flow) may be called in several ways:
+
++ by referencing the [operation](#operation) or [flow](#flow) by name when it is in the default namespace (the same namespace as the calling [flow](#flow))
++ by using a fully qualified name (e.g. `path.to.operation.op_name`)
```

Comment:
maybe specify here that a path in only recognized as full path if the the prefix `path` (first word before `.` character) is not a defined alias (cannot be expanded)
WDYT?